meetings/2021-11-23.md

```diff
@@ -0,0 +1,115 @@
+
+* Pavlik: Great use case - haven't given it a ton of thought yet. From a resource standpoint we always expect to get the social agent and the client (application). Need to be careful how we expect those pairings to get pulled together. The scenario presented there are two cases: 
+- Server-side service used as an application by the user (e.g. user logs in and the user's webid is leveraged, and app needs to keep some state to know which to send for which user). Would be a different instance of fetch per user
+- Less explored - if the user is giving access to the organization the runs the application and the application is running "headless", when requests are made to the storage, the "organization" would appear as the social agent with the client identifier. Need to clarify who is being authorization - organization vs. user, and whose webid is presented.
+
+* Justin: I agree with how you laid out the two scenarios. It would be probably fine and appropriate to maintain the pairing as organization as social agent and application, when it operates in this 'headless' way. In the past the we were using same uri for webid and client_id. In application profile we already identify the responsible social agent as author.
+* ...: We have to detail out a little bit what the moving parts look on the authorization part on Solid-OIDC. Actually that's the first use case I need to demonstrate with Java library. I can take action item to dig into this one.
+
+* Pavlik: if you could focus on the consent screen presented to the user, do they have an application registration, or is it a social agent registration? Possibly there will be cases for both. Possibly approach it in both ways.
+* Justin: My gut says that you would want to have Application Registration, but they could be case where we want to have Social Agent registration for the organization. I may want to authorize social agent services one by one.
+
+* Eric: If there is pairing between 
+
+* Pavlik: When saying the user wants to authorize certain services, we could use access need groups for that. Services could be expressed that way. For different services there could be different access need groups.
+
+* Justin: I think there is good reason to have both available. Organization could use access need groups under application umbrella. It's nice to have that flexibility
+
+### 
+
+* Eric: We can extend grant in the future that if grant has delegation privilege they can pass it further. As proof-carrying delegation. It's dynamic access control, owner doesn't maintain list of who has access.
+
+* Pavlik: Lines up with where i"m hoping to head with delegated / trusted grants. Already have setup for delegated grants between owner and application they use. Can use the same pattern in a chain. Also need to have an upstream notification of delegation. Omni gives Alice access. Alice delegates that access to Bob. Alice needs to let Omni know so it can update its records. Will share a link to authz panel. 
```

Comment:
```suggestion
* Pavlik: Lines up with where i"m hoping to head with delegated / trusted grants. Already have setup for delegated grants between owner and application they use. Can use the same pattern in a chain. Also need to have an upstream notification of delegation. Omni gives Alice access. Alice delegates that access to Bob. Alice needs to let Omni know, so it can update its records. Will share a link to authz panel. 
```